add v1.2.0 documentation

sandro-elsweijer opened the pull request.

sandro-elsweijer committed:
add v1.2.0 documentation

holke reviewed (changes requested):
The subfolders of src/ are missing, i.e. src/t8_cmesh etc.

sandro-elsweijer committed:
add subfolder documentation

sandro-elsweijer commented:
> The subfolders of src/ are missing, i.e. src/t8_cmesh etc.

I changed the doxygen recursive option locally and added the subfolder documentation as well

sandro-elsweijer committed:
now with the cpp files
